packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm

```diff
@@ -98,6 +98,9 @@ @implementation RCTScrollViewComponentView {
   // some other part of the system scrolls scroll view.
   BOOL _isUserTriggeredScrolling;
   BOOL _shouldUpdateContentInsetAdjustmentBehavior;
+  BOOL _automaticallyAdjustKeyboardInsets;
+  BOOL _inverted;
+  UIEdgeInsets _contentInset;
```

Comment:
Can't we just use the `scrollView.contentInsets` which are the source of truth for `UIScrollView`?

Reply:
Sure we can! I missed it, fixed 👍🏻